recipes/nds2-client-swig/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
I just would recommend you to create a build folder and build everything there.
I would recommend you to do
```
mkdir -p build & cd build
```

and in the ``cmake`` command you just have to put ``..`` (tow dots) after the commando, something like

```
cmake .. \
```
```

Reply:
@marcelotrevisani, what woukd be the benefit of this? I don't really object but think fewer steps will be easier to maintain.

Reply:
It is customary not to mix the built files with the source. Just better practice.

Reply:
@moorepants, thanks, I implemented this in a8e3dd9